Add a new blogs page and v2.3.0 release

arcturus2 opened the pull request:
* Minor update to order of content on the home page
* Updated the links to the download page to go directly to the download
  page
* Added a blogs page
* Added the first blog which is the release of v2.3.0

arcturus2 committed:
:memo: Add a new blogs page and v2.3.0 release
* Minor update to order of content on the home page
* Updated the links to the download page to go directly to the download
  page
* Added a blogs page
* Added the first blog which is the release of v2.3.0

arcturus2 commented:
Attached is the first preview based on the commit so far. I will be working on the rest of the images now. If you download and extract the zip file you should be able to click on the index.html page and have a look.

[site-v3.zip](https://github.com/constellation-app/constellation-app.github.io/files/6572943/site-v3.zip)

arcturus2 committed:
:memo: Add the download link to the home page

arcturus2 committed:
:memo: Added pending images and updates
* Removed excessive hyperlinks that don't go anywhere
* Added images to the blog

arcturus2 committed:
:memo: Add a reference to Cyber v1.2

arcturus2 commented:
Attached is the latest version including the pending images and some feedback.

[site-v4.zip](https://github.com/constellation-app/constellation-app.github.io/files/6573486/site-v4.zip)

arcturus2 committed:
:memo: Make the reference to the word Constellation consistent case

antares1470 commented on docs/download.html:
```diff
@@ -73,9 +74,9 @@ <h3>Download Constellation</h3>
                                 <li>Constellation Adapters</li>
```
Adaptors

arcturus2 replied:
Fixed thank you very much for picking up on the typo

aldebaran30701 commented:
Love the look of the blog! Loads of progress.
One thing I found a bit odd was the fact that you click 'read more' to view the rest of the blog. It may be useful for when many blogs are listed one under another - but it just felt confusing for me. Totally okay to keep it as is, just a small mention of it.
Couldn't see anything to pick on it - gave the whole blog page a good read.

sol695510 commented:
![image](https://user-images.githubusercontent.com/58677774/120281517-31e98e80-c2f8-11eb-8dbd-e0d66a69ecf6.png)

I think it looks great for the most part, but I think that the title here needs to be more prominent, since the background has a lot going on (which I think is fine). As labelled in the screenshot:
1. I think this title could just be made bigger.
2. This line/divider could be made "bold" because it blends into the background a bit as it is.
3. This could be made bigger and possibly "bold" as well, but up to you since its a subtitle.

As it is, this component takes up about a 1/3 of this section of background, whereas I think it should be between 1/2 to 2/3 to increase its readability.

sol695510 commented:
Current:
![placeholder](https://user-images.githubusercontent.com/58677774/120284239-38c5d080-c2fb-11eb-990a-3dbb7e7c955f.png)

Proposed:
![Recent_Updates_Logo_Idea](https://user-images.githubusercontent.com/58677774/120283879-d240b280-c2fa-11eb-8d77-7ffb03722fef.png)

This is just a rough design, but I think the Recent Updates carousel would look better with some version text on the icon for each linked version, not too much text but some key text for quick visual recognition.

arcturus2 committed:
:memo: Updates based on feedback
* Fix to spelling
* Improvements to gramma
* Reduce image size using tinypng.com saving a tone of space overall
* Fix to copyright reference on the index page
* Removed the image above the "Recent Updates" section until there is
  time to make them all unique
* Changed the version name to v2.3 ommitting the .0 at the end as its
  not useful visually. The links however will have this for consistency

arcturus2 commented:
> I think it looks great for the most part, but I think that the title here needs to be more prominent, since the background has a lot going on (which I think is fine). As labelled in the screenshot:
> 
>     1. I think this title could just be made bigger.
> 
>     2. This line/divider could be made "bold" because it blends into the background a bit as it is.
> 
>     3. This could be made bigger and possibly "bold" as well, but up to you since its a subtitle.
> 
> 
> As it is, this component takes up about a 1/3 of this section of background, whereas I think it should be between 1/2 to 2/3 to increase its readability.

Yeah that is decent feedback. I have not been able to make this change today but will consider doing so tomorrow

arcturus2 commented:
> This is just a rough design, but I think the Recent Updates carousel would look better with some version text on the icon for each linked version, not too much text but some key text for quick visual recognition.

Totally agree with this so for now I've removed the images all together to stop the repetition. I was looking at an alternate design by putting the version number at an angle to one of the corners but started spending too much time I stopped for now.

arcturus2 committed:
:memo: Fix to missed Constellation capitalisation
:art: Reduce the file size of more images

arcturus2 committed:
:art: Update the primary and secondary colours

arcturus2 commented:
The latest site with some suggestions made. Others will be pushed back to next time.

[site-v5.zip](https://github.com/constellation-app/constellation-app.github.io/files/6581139/site-v5.zip)

arcturus2 commented:
@sol695510 i created https://github.com/constellation-app/constellation/issues/1078 to track your suggestions for next time. Running out of time to fix them now.

arcturus2 committed:
:memo: Add a download button to the blog page

arcturus2 committed:
:art: Remove redundant hyperlinks to images
